docs/user-guide/kubectl-overview.md

```diff
@@ -266,7 +266,29 @@ $ kubectl logs <pod-name>
 $ kubectl logs -f <pod-name>
 ```
 
+## Using kubectl with multiple remote environments
+
+When managing multiple Kubernetes clusters, it is very useful to be able to run kubectl on your local machine, and connect to different remote Kubernetes installations. First of all, [download and install](http://kubernetes.io/docs/getting-started-guides/docker/#download-kubectl) kubectl on your machine.
```

Comment:
Not sure what the best place to point to is. The binary_release is a larger download than just kubectl. On the other hand there are instructions on that page for Mac users to use homebrew. I'll change the link like you suggested.